src/runtime/entitlements-manager.js

```diff
@@ -88,6 +88,9 @@ export class EntitlementsManager {
     /** @private {boolean} */
     this.blockNextNotification_ = false;
 
+    /** @private {?string} */
```

Comment:
Could you add a comment that mentions currently this is used for metering params but we might expand it in the future?

Reply:
Done.